tests/codebuild/run_integtest.sh

```diff
@@ -73,10 +73,10 @@ set -e
 if [ "${need_setup_cluster}" == "true" ]; then
     echo "Launching the cluster"
     readonly cluster_name="sagemaker-k8s-pipeline-"$(date '+%Y-%m-%d-%H-%M-%S')""
-    readonly cluster_region="us-east-1"
```

Comment:
We should probably keep this line here until `cluster_region` is removed from the `need_setup_cluster != true` case. 

I only see [one (currently disabled) usage](https://github.com/aws/amazon-sagemaker-operator-for-k8s/search?q=cluster_region&unscoped_q=cluster_region) in integration tests, but the fact that this is disabled will be difficult to know for someone unfamiliar with the code.

Reply:
Sure. I will reassign this variable for use within integration tests. I didn't realise we were passing it around.